services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -469,6 +456,57 @@ public LinkedAccountsResource linkedAccounts() {
         return new LinkedAccountsResource(session, request, client, auth, event, user);
     }
 
+    @Path("/applications")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @NoCache
+    public Response applications() {
+        checkAccountApiEnabled();
+        auth.require(AccountRoles.VIEW_APPLICATIONS);
```

Comment:
@douglaspalmer not sure why it was removed, but this endpoint used to have the following permissions:
```
auth.requireOneOf(AccountRoles.VIEW_APPLICATIONS, AccountRoles.VIEW_PROFILE);
```
Without that, the UI will get HTTP 403.